unit-tests/src/org/commcare/android/tests/processing/FormStorageTest.java

```diff
@@ -26,12 +31,187 @@
 @RunWith(CommCareTestRunner.class)
 public class FormStorageTest {
 
+    private static final List<String> classNames =
```

Comment:
I assume this is just a list you compiled manually of classes that implement `Externalizable`?